src/tracking.F90

```diff
@@ -498,8 +498,8 @@ subroutine cross_surface(p)
       ! set new cell value
       p % coord(1) % cell = i_cell-1 ! decrement for C++ indexing
       p % cell_instance = 1
-      p % material = cells(i_cell) % material(1)
-      p % sqrtKT = cells(i_cell) % sqrtKT(1)
+      p % material = cells(i_cell) % material(0)
+      p % sqrtKT = cells(i_cell) % sqrtKT(0)
```

Comment:
Strange, I recently had to make this change to avoid segfaults caused by indexing beyond the size of those vectors... I'll look into it more closely and get back to you.

Reply:
Got it. I added a thermal xs to the `dagmc` test to ensure some of the cell/material initialization was working properly. I found that the temperature in one of the cells wasn't being looked up correctly and changed both of these values assuming that the methods would have the same indexing behavior. 

Long story, short: `cell_material` has a -1, but `cell_sqrtkT` does not.

Pushing a fix and updated test results for this soon. Thanks for addressing that.